database/seeders/UsersTableSeeder.php

```diff
@@ -19,7 +19,7 @@ public function run()
             //Name, email, password, type
             'name' => 'Alex Legard',
             'email' => 'alexlegard3@gmail.com',
-            'password' => '$2y$10$qDsvuA4OPejyJqBXFzAcXuhp1fc1bABTvoIuH9ytXyFlor.GgcBRe',
+            'password' => \Hash::make('password'),
```

Comment:
Same as above

Reply:
Will this code simply set the password to 'password'?

Reply:
Yup - obviously it's not a safe one, but it allows people to login to the site after seeding.